src/platform/stm32f1/spi.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * SPDX-License-Identifier: MIT
+ * SPDX-FileCopyrightText: 2021 Rafael Silva <silvagracarafael@gmail.com>
+ */
+
+#pragma once
+
+#include <stm32f1xx.h>
+
+#include "platform/stm32f1/gpio.h"
+
+#define SPI_MSB_FIRST 0
+#define SPI_LSB_FIRST 1
```

Comment:
Why `{LSB,MSB}_FIRST` instead of `{LITTLE,BIG}_ENDIAN`? Or is it not common to refer them that way?

Reply:
`{LSB,MSB}_FIRST` is the common way this is refered to, not that `{LITTLE,BIG}_ENDIAN` wouldn't work here

Reply:
Let's use the common way then.